en/docs/references/pricing.md

```diff
@@ -0,0 +1,229 @@
+
+## Plans
+
+|               | Free                   |  Pay As You Go    | Enterprise     |
+|---------------|------------------------|-------------------|----------------|
+| Cost Per Step | 5,000 steps per month *|$5 per 1,000 steps <br/> <ul><li>First 5,000 steps per month are free</li></ul>| Contact us for discounted pricing including: <br/><ul><li> Volume Discounts</li><li> Committed-use Discounts</li><li> Support Channels and Support SLA</li></ul>|
+
+ \* Limited to a maximum of 3 components
+
+
+
+## Choreo pricing FAQ
+
+### How does Choreo pricing work?
+
+Choreo pricing is based on the number of [steps](#what-is-a-step) in your integration solution. Choreo offers three payment plans: Free, Pay As You Go, and Enterprise.
+
+#### Free 
+The Choreo free tier allows you to execute 5,000 [steps](#what-is-a-step) per month, limited to a maximum of 3 components. If you exceed the free tier, your application cannot make any additional requests to Choreo. 
+
+#### Pay As You Go (PAYG) 
+Pay As You Go is a flexible usage-based payment plan. While the first 5,000 steps are free per month, Choreo will charge you $0.005 for each additional step, for example, $5 for 1,000 steps based on your usage per month. 
+
+Let’s say you have used a total of 12,500 [steps](#what-is-a-step) for a particular month, your monthly usage for that month under the PAYG plan will be:
+
+    Chargeable steps usage: 12,500 - 5,000 = 7,500
+    The total charge for the usage: 7,500 * $0.005 ($5 for every 1,000 steps) = $37.50
+
+#### Enterprise 
+The Enterprise plan is suitable for large-scale usage of Choreo. You can access volume discounts and committed-use discounts with this plan. [Contact sales](https://wso2.com/contact/?ref=choreo) to get a discounted rate.
+
+### What is a step?
+A step (1 step) is a maximum of 500ms compute time on a container having 350MB of memory and 0.5 vCPU. Any additional compute time is counted as an additional step.  
+An incoming event or message to Choreo or an outgoing API call from Choreo is a minimum of 1 step.
+
+
+## Pricing examples
+
+### Example 1: REST API 
+
+An organization has a flight booking system on Choreo. This system involves several APIs and databases. The scenario and the APIs being used in the system are described below.
+
+#### Scenario
+
+To make a booking through the web application, a client will first search for available flights and then proceed to make the booking. The client application (web application) upon receiving the request will make a call to the [Search API](#searchAPI) to search for available flights. When the client receives the search results the client will proceed to make a reservation using the [Reservations API](#reservationsAPI).
```

Comment:
In this context, present tense is more appropriate?

```suggestion
To make a booking through the web application, a client first searches for available flights and then proceeds to make the booking. The client application (i.e., the web application) upon receiving the request makes a call to the [Search API](#searchAPI) to search for available flights. When the client receives the search results, the client proceeds to make a reservation using the [Reservations API](#reservationsAPI).
```

Reply:
done